prog/vm/nexus.rb

```diff
@@ -224,17 +213,13 @@ def create_storage_volume_records(vm_host)
         )
       end
 
-      spdk_installation_id =
-        allocate_spdk_installation(
-          vm_host.spdk_installations,
-          use_bdev_ubi: volume["use_bdev_ubi"]
-        )
+      spdk_installation_id = allocate_spdk_installation(vm_host.spdk_installations)
 
       VmStorageVolume.create_with_id(
         vm_id: vm.id,
         boot: volume["boot"],
         size_gib: volume["size_gib"],
-        use_bdev_ubi: volume["use_bdev_ubi"],
+        use_bdev_ubi: SpdkInstallation[spdk_installation_id].supports_bdev_ubi?,
```

Comment:
this should also be `false` for non-boot disks, since bdev_ubi requires a base image. Currently, we only use base image for boot disks. So, `&& !volume["boot"]`.